docs/src/main/resources/html/header.include

```diff
@@ -25,7 +26,7 @@
 	<section id="pageheader" class="pb-sm-0 pb-md-0 pb-lg-5 pt-0">
 		<div class="container">
 			<div class="row">
-				<h1 id="headertext" class="mt-5"><i class="fa fa-file-o fa-fw"></i>titleToken</h1>
+				<h1 id="headertext" class="mt-5"><i class="fab fa-java fa-fw"></i>titleToken</h1>
```

Comment:
Why has this changed? I didn't think these classes had changed, and it probably means it will not work once it's in the OSH website. Or is this another set of icons all-together? If so, is there really a need for another set?

Reply:
> Why has this changed? I didn't think these classes had changed, and it probably means it will not work once it's in the OSH website.

I realized that last night. The website is using FontAwesome 4 mininal and I needed some added icons. When getting the new link I moved to the current FontAwesome 5. Version 6 will be released soon.

I plan on moving back to FontAwesome 4. to insure the move does not break on the website If the Java icon is not available in FontAwesome 4 I will likely use an iimage file.

I am quickly getting to the place where it would be good to see the documentation in a test area of the website to insure compatibility. Using maven to compile the docs pointed out MarkDown differences from the GitHub rendering and I have been addressing those.

I view this as an iterative process, improving as the features evolve.